eslzArm/eslzArm.json

```diff
@@ -607,8 +607,7 @@
             "ascConfigPolicyInitiative": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-ASCConfigPolicyAssignment.json')]",
             "azVmMonitorPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-VMMonitoringPolicyAssignment.json')]",
             "azVmssMonitorPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-VMSSMonitoringPolicyAssignment.json')]",
-            "azBackupLzPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-VMBackupPolicyAssignment.json')]",
-            "azBackupIdentityPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-VMBackupPolicyAssignment.json')]",
+            "azVmBackupPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-VMBackupPolicyAssignment.json')]",
             "azPolicyForAksPolicyAssignment": "[uri(deployment().properties.templateLink.uri, 'managementGroupTemplates/policyAssignments/DINE-AksPolicyPolicyAssignment.json')]",
```

Comment:
If identity subscription is enabled, we also recommend to assign VM Backup policy to the identity management group, hence we need a deterministic guid for the subsequent role assignment for that policy, so we need both these variables.

Reply:
As `azBackupIdentityPolicyAssignment` isn't referenced anywhere else in the repository and the values are the same, how does this make a difference when an identity subscription is enabled?

Reply:
It is it's own (optional) deployment name, and we cannot have duplicated resource name in the same ARM template.

Reply:
Just my 2 cents here 😃 

I have tested a deployment from @krowlandson's branch for this PR and it works 👍 
![image](https://user-images.githubusercontent.com/41163455/128745946-01267be9-1824-43ce-bace-d139dfca2ba1.png)

Including the Backup policies involved also:
![image](https://user-images.githubusercontent.com/41163455/128746008-3a49248f-18bd-40ae-96a4-eaba5819c095.png)

From my initial review, and re-review today, the ARM deployment names are unchanged just the reference to the deployment template URI and a de-duplication of a variable that holds this (which from looking at the current templates, it looks like the `azBackupIdentityPolicyAssignment` variable is not referenced anywhere as the `azBackupLzPolicyAssignment` is used for all nested deployments which do the assignments. Also as the conditionals are not based on these and other parameters/variables from the portal UX experience then I think this is all good 👍